### ISSUE TITLE
Exception is thrown when reading tags from an image

### ISSUE THREAD

**Issue by zdila** [mattiasw/ExifReader#248]
### Description

Exception is thrown when reading tags from an image.

It seems to be caused by reading some thumbnail data. I' ve tried to disable thumbnails with custom exif configuration, but according to your webpack config the string replacement regex doesn't cover `tags.js` file (this should be fixed too).

**Additional details**

- ExifReader version: 4.13.0
- Web browser and version: 6.2.3105.51 (Stable channel) stable (64-bit)
- Node version: v18.18.0

**How to reproduce**

1.  extract image from [image.zip](https://github.com/mattiasw/ExifReader/files/12776713/image.zip)
2.  try to read tags

**What I expected would happen:**

Thags would be read.

**What really happened:**

Exception is thrown:

```
RangeError: Maximum call stack size exceeded
    at Object.getAsciiValue (types.js:47:1)
    at getTagValue (tags.js:324:1)
    at readTag (tags.js:268:1)
    at readIfd (tags.js:218:1)
    at readIfd (tags.js:233:1)
    at readIfd (tags.js:233:1)
    at readIfd (tags.js:233:1)
    at readIfd (tags.js:233:1)
    at readIfd (tags.js:233:1)
    at readIfd (tags.js:233:1)
```

**Comment by mattiasw** [comment]
Hi! Thanks for the report. I've fixed the missing `tags.js` file in the replacement regex in the latest commit. If the other error is taking too long to fix I'll release that by itself in the meantime.

**Comment by mattiasw** [comment]
Thumbnail exclusion fix released as version 4.13.1. I'll keep working on the real error. I have something that seems to work but need to understand why. 😅

**Comment by zdila** [comment]
Replacement now works but my setup actually reads source files directly so I had to add rule to my own webpack config:
```
{
        test: /\/exifreader\/src\/tags\.js$/,
        loader: 'string-replace-loader',
        options: {
          search: 'Constants.USE_THUMBNAIL',
          replace: 'false',
        },
      },
```

**Comment by mattiasw** [comment]
That's a shame. There is a suggestion for helping with that situation and it would be a great addition but that will be some time in the future: https://github.com/mattiasw/ExifReader/issues/243

**Comment by mattiasw** [comment]
A real fix for this is now released as version 4.13.2. Thanks for reporting. After the fix another two of my test files produced thumbnails that the library hadn't found before.
